system-int-test/test/cmr/system_int_test/search/granule_counts_search_test.clj

```diff
@@ -752,3 +753,38 @@
 
               "granule count in json format"
               :atom (search/find-concepts-json :collection {:include-granule-counts false})))))
+
+
+(deftest over-max-int-granule-hits-test
```

Comment:
Remove or comment out this test. We may want to manually run a test to verify that we can go over the `Integer/MAX_VALUE` limit, but we certainly don't want this test in our integration test suite.